esd_services_api_client/crystal/_models.py

```diff
@@ -143,4 +154,5 @@ class AlgorithmRequest(DataClassJsonMixin):
     algorithm_parameters: Dict
     algorithm_name: Optional[str] = None
     custom_configuration: Optional[AlgorithmConfiguration] = None
+    parent_job: Optional[ParentJob] = None
```

Comment:
```suggestion
    parent_request: Optional[ParentJob] = None
```
let's keep naming consistent - job is more internal Crystal jargon. However, you do not need this rn - just rename before rolling the feature out